File: test/fixtures/output/node/request/jsonObj-multiline.js
var request = require("request");

var options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {'content-type': 'application/json'},
  body: {foo: 'bar'},
  json: true
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/ruby/faraday/fixtures/jsonObj-multiline.rb
require 'faraday'

conn = Faraday.new(
  url: 'http://mockbin.com',
  headers: {'Content-Type' => 'application/json'}
)

response = conn.post('/har') do |req|
  req.body = "{\n  \"foo\": \"bar\"\n}"
end

puts response.status
puts response.body